src/PyCall.jl

```diff
@@ -298,6 +300,14 @@ function setindex!(o::PyObject, v, s::Union{Symbol,AbstractString})
     o
 end
 
+getproperty(o::PyObject, s::Symbol) = getindex(o, s)
+getproperty(o::PyObject, s::AbstractString) = getindex(o, s)
+
+propertynames(o::PyObject) = map(x->Symbol(first(x)), 
+                                pycall(inspect["getmembers"], PyObject, o))
+
+setproperty!(o::PyObject, s::Union{Symbol,AbstractString}, v) = setindex!(o, v, s)
+
 function haskey(o::PyObject, s::Union{Symbol,AbstractString})
```

Comment:
We'll need to change this to `hasproperty` or something?  Not sure if there is a Julia `Base` function to override here.

Reply:
(https://github.com/JuliaLang/julia/issues/28850)

Reply:
I'll rename it to `hasproperty` everyhere. We can override the `Base` function once it's there.